docs/usage/list.rst

```diff
@@ -32,3 +32,7 @@ Examples
     drwxrwxr-x user   user          0 Sun, 2015-02-01 11:00:00 code/myproject
     -rw-rw-r-- user   user    1416192 Sun, 2015-02-01 11:00:00 code/myproject/file.ext
     ...
+
+    $ borg list /path/to/repo/::archiveA --pattern "re:ext$"
```

Comment:
`'re:\.ext$'` would make sense:

- shows how to escape a dot (please check whether \ or \\ is needed)
- uses single quotes, so the shell does not expand anything
- makes sure only .ext extension matches and not .text also

Reply:
Should be good now. As far as I can tell both `\` and `\\` have the same effect.